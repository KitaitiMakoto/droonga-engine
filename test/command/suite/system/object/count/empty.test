{
  "type": "system.object.count",
  "dataset": "Default",
  "body": {
    "output": [
      "tables",
      "columns",
      "records",
      "total"
    ]
  }
}
