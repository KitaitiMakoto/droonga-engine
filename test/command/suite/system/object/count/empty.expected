{
  "inReplyTo": "request-id",
  "statusCode": 200,
  "type": "system.object.count.result",
  "body": {
    "tables":  0,
    "columns": 0,
    "records": 0,
    "total":   0
  }
}
